src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -12112,6 +12113,71 @@ class ClrStackImpl
+        CROSS_PLATFORM_CONTEXT context;
+        HRESULT hr = pStackWalk->GetContext(DT_CONTEXT_FULL, g_targetMachine->GetContextSize(), NULL, (BYTE *)&context);
+        if (FAILED(hr) || hr == S_FALSE)
+        {
+            // GetFrameContext returns S_FALSE if the frame iterator is invalid.  That's basically an error for us.
+            ExtOut("GetFrameContext failed: %lx\n", hr);
+            return E_FAIL;
+        }
+                     
+#if defined(SOS_TARGET_AMD64)
+        String outputFormat3 = "    %-3s=%016x %-3s=%016x %-3s=%016x\n";
+        String outputFormat2 = "    %-3s=%016x %-3s=%016x\n";
+        String outputFormat1 = "    %-3s=%016x\n";
+        ExtOut(outputFormat3, "rax", context.Amd64Context.Rax, "rbx", context.Amd64Context.Rbx, "rcx", context.Amd64Context.Rcx);
+        ExtOut(outputFormat3, "rdx", context.Amd64Context.Rdx, "rsp", context.Amd64Context.Rsp, "rbp", context.Amd64Context.Rbp);
+        ExtOut(outputFormat3, "rip", context.Amd64Context.Rip, "rsi", context.Amd64Context.Rsi, "rdi", context.Amd64Context.Rdi);
+        ExtOut(outputFormat3, "r08", context.Amd64Context.R8, "r09", context.Amd64Context.R9, "r10", context.Amd64Context.R10);
+        ExtOut(outputFormat3, "r11", context.Amd64Context.R11, "r12", context.Amd64Context.R12, "r13", context.Amd64Context.R13);
+        ExtOut(outputFormat2, "r14", context.Amd64Context.R14, "r15", context.Amd64Context.R15);
+        ExtOut(outputFormat3, "cs", context.Amd64Context.SegCs, "ds", context.Amd64Context.SegDs, "es", context.Amd64Context.SegEs);
+        ExtOut(outputFormat3, "fs", context.Amd64Context.SegFs, "gs", context.Amd64Context.SegGs, "ss", context.Amd64Context.SegSs);
+        ExtOut(outputFormat1, "efl", context.Amd64Context.EFlags);
+#elif defined(SOS_TARGET_X86)
+        String outputFormat3 = "    %-3s=%08x %-3s=%08x %-3s=%08x\n";
+        String outputFormat2 = "    %-3s=%08x %-3s=%08x\n";
+        String outputFormat1 = "    %-3s=%08x\n";
+        ExtOut(outputFormat3, "eax", context.X86Context.Eax, "ebx", context.X86Context.Ebx, "ecx", context.X86Context.Ecx);
+        ExtOut(outputFormat3, "edx", context.X86Context.Edx, "esp", context.X86Context.Esp, "ebp", context.X86Context.Ebp);
+        ExtOut(outputFormat3, "eip", context.X86Context.Eip, "esi", context.X86Context.Esi, "edi", context.X86Context.Edi);
+        ExtOut(outputFormat3, "cs", context.X86Context.SegCs, "ds", context.X86Context.SegDs, "es", context.X86Context.SegEs);
+        ExtOut(outputFormat3, "fs", context.X86Context.SegFs, "gs", context.X86Context.SegGs, "ss", context.X86Context.SegSs);
+        ExtOut(outputFormat1, "efl", context.X86Context.EFlags);
+#elif defined(SOS_TARGET_ARM)
+        String outputFormat3 = "    %s=%08x %s=%08x %s=%08x\n";
+        String outputFormat2 = "    %s=%08x %s=%08x\n";
+        String outputFormat1 = "    %s=%08x\n";
+        ExtOut(outputFormat3, "r00", context.ArmContext.R0, "r01", context.ArmContext.R1, "r02", context.ArmContext.R2);
```

Comment:
I don't know about how most debuggers display ARM registers but we should probably match how LLDB displays them.